third_party/uncrustify.cfg

```diff
@@ -719,7 +719,7 @@ indent_namespace_limit          = 0        # unsigned number
 indent_extern                   = false    # false/true
 
 # Whether the 'class' body is indented.
-indent_class                    = false    # false/true
+indent_class                    = true    # false/true
```

Comment:
This is also usually set to `false` in most C++ repos. Commit 92d0e124ae4f79330f222974a225eb1e08e54e63

Reply:
IMHO, methods should indent in class, either 2 spaces or 4 spaces because it should be more readable.
@OpenGenus/cosmos-maintainers What is your opinion on this?

Reply:
@iattempt The class itself should be indented 4 spaces, **NOT** the access specifiers (`public`, `private`, `protected`).

Reply:
@arnavb is this what you mean for?
```
class Tree
{
    class Node
    {
public:  <----
        void value(int v)
        {
            // do something
        }
    }
}
```

Reply:
@iattempt This is what I mean:

```C++
class Tree
{
private: <---
    class Node
    {
    public:  <----
        void value(int v)
        {
            // do something
        }
    }
}
```